lib/builder/generator.js

```diff
@@ -73,11 +73,14 @@ export default class Generator extends Tapable {
       }))
     }
 
-    // Copy /index.html to /200.html for surge SPA
-    // https://surge.sh/help/adding-a-200-page-for-client-side-routing
-    const _200Path = join(this.distPath, '200.html')
-    if (!existsSync(_200Path)) {
-      await copy(join(this.distPath, 'index.html'), _200Path)
+    const indexPath = join(this.distPath, 'index.html')
+    if (existsSync(indexPath) {
```

Comment:
missing a `)` here

Reply:
oops :-1:

thanks for spotting that

Reply:
Opened a pr to fix : https://github.com/nuxt/nuxt.js/pull/1950

Reply:
no idea how i missed that, same issue is in nuxt-generate-cluster and I worked on that all yesterday evening without noticing :confused:

i will go stand in the corner

Reply:
Don't worry, CI can help us to find out it 😸 .